nameko/testing/utils.py

```diff
@@ -89,6 +90,13 @@ def assert_stops_raising(fn, exception_type=Exception, timeout=10,
             eventlet.sleep(interval)
 
 
+MockCallArgs = namedtuple("MockCallArgs", ["positional", "keyword"])
+
+
+def unpack_mock_call(call):
```

Comment:
@tonybajan is that an endorsement for `call` being a namedtuple or for the helper?